src/Rfc6455/MessageHandler/PingFrameHandler.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * This file is a part of Woketo package.
+ *
+ * (c) Ci-tron <dev@ci-tron.org>
```

Comment:
OMG I copied a class of woketo and there are  throughout the project... You should also look at the configuration of your PhpStorm XD. I correct the ci-tron in my commit?